src/chunk_list.h

```diff
@@ -797,6 +798,16 @@ static_inline bool chunk_is_forin(chunk_t *pc)
 }
 
 
+/**
+ * Checks if a chunk is a keyword
+ *
+ * @return true  - the chunk is keyword
+ */
+static_inline bool chunk_is_keyword(chunk_t *pc)
```

Comment:
I don't see function declaration in `keywords.h`. I think it can be useful to put it there as well.